### PR TITLE
Adding com.github.louis77.tuner

### PR DESCRIPTION
**Pull request by louis77**
I'm the author of this App. Would be great to have it added to Flathub!

### CODE REVIEW

**Comment by louis77** [comment]
bot, build com.github.louis77.tuner

File: com.github.louis77.tuner.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "com.github.louis77.tuner",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.louis77.tuner",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
```
Comment (bilelmoussaoui) [inline com.github.louis77.tuner.json:10]:
```suggestion
        "--share=ipc", "--socket=fallback-x11",
```

File: com.github.louis77.tuner.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        "--socket=wayland",
+        "--filesystem=host",
```
Comment (bilelmoussaoui) [inline com.github.louis77.tuner.json:12]: What does it need host access for?

File: com.github.louis77.tuner.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--share=network",
+        "--metadata=X-DConf=migrate-path=/com/github/louis77/tuner/",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+        "--talk-name=org.mpris.MediaPlayer2",
```
Comment (bilelmoussaoui) [inline com.github.louis77.tuner.json:20]:
```suggestion
```
do you actually need this one?

File: com.github.louis77.tuner.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/taglib/taglib/archive/v1.11.1.tar.gz",
+                "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+            }]
+        },
+        {
+            "name": "tuner",
+            "buildsystem": "meson",
+            "build-options": {
```
Comment (bilelmoussaoui) [inline com.github.louis77.tuner.json:52]: there's a config-opts key as well, you don't need to set build-options to set them.

File: com.github.louis77.tuner.json (hunk 2 above)
Comment (louis77) [reply com.github.louis77.tuner.json]: It didn't, thank you for the suggestion!

File: com.github.louis77.tuner.json (hunk 3 above)
Comment (louis77) [reply com.github.louis77.tuner.json]: Removed it

File: com.github.louis77.tuner.json (hunk 4 above)
Comment (louis77) [reply com.github.louis77.tuner.json]: Thank you for the hint!

**Comment by louis77** [comment]
bot, build com.github.louis77.tuner

**Comment by bilelmoussaoui** [comment]
There are some missing icons in the sidebar, in general most of the elementary apps have their icons/stylesheets hardcoded to ensure they work correctly. Otherwise looks good to me

**Comment by louis77** [comment]
bot, build com.github.louis77.tuner

**Pull request closed without merging**